lib/_http_incoming.js

```diff
@@ -93,21 +93,17 @@ IncomingMessage.prototype.destroy = function(error) {
 
 IncomingMessage.prototype._addHeaderLines = function(headers, n) {
   if (headers && headers.length) {
-    var raw, dest;
+    var dest;
     if (this.complete) {
-      raw = this.rawTrailers;
+      this.rawTrailers = headers;
```

Comment:
I don't know why a separate array was being created in this function. A new array is already created for every parse of an incoming message, so this particular change just holds onto that reference instead of having it go unused and being GC'ed after parsing.